charts/argo-workflows/README.md

```diff
@@ -6,6 +6,12 @@ If you want your deployment of this helm chart to most closely match the [argo C
 
 ## Pre-Requisites
 
+### Supported Versions
+- Kubernetes: `>=1.23.0-0`
+  - We align with [Amazon EKS calendar][EKS EoL] because there are many AWS users and it's a conservative approach.
```

Comment:
Aligning to EKS is the most conservative.

- EKS
https://endoflife.date/amazon-eks
<img width="783" alt="image" src="https://github.com/argoproj/argo-helm/assets/30188755/2cc83f1d-5d46-49d6-b613-c0a211156e27">


- K8s itself
https://endoflife.date/kubernetes
<img width="814" alt="image" src="https://github.com/argoproj/argo-helm/assets/30188755/ffe2265f-ad3d-484c-a8cf-586063be7693">

- GKE
https://endoflife.date/google-kubernetes-engine
<img width="837" alt="image" src="https://github.com/argoproj/argo-helm/assets/30188755/6e3f5887-a6b9-4bb9-a15f-3c8abf5839bf">

- AKS
https://endoflife.date/azure-kubernetes-service
<img width="801" alt="image" src="https://github.com/argoproj/argo-helm/assets/30188755/a4584e8e-e7c1-40f9-9bdb-5e20f52e31b3">